config-common/src/watcher.rs

```diff
@@ -55,10 +43,10 @@ pub fn start_watcher<TConfig: 'static, TError: 'static>(
             // Add configuration paths to be watched.
             file_watcher
```

Comment:
This needs a check for whether the `config_directory_path` exists. It may not. Eg `read_config` treats `std::io::ErrorKind::NotFound` as success when discovering config files.

Reply:
Fixed.